dev-haskell/beam-automigrate/beam-automigrate-0.1.2.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="DB migration library for beam, targeting Postgres"
+HOMEPAGE="https://hackage.haskell.org/package/beam-automigrate"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="ghcipretty integration-tests werror examples"
```

Comment:
I doubt we actually need the `integration-tests` `USE`-flag, it looks like it requires network access during the tests.

edit: yea, I tried it with `USE=integration-tests` and the tests fail. without it, the tests pass. I would remove this `USE`-flag entirely, as well as the `tmp-postgres` dependency, and then make sure to `--flag=-integration-tests` in `src_configure`

Reply:
Removed, as mentioned in a comment above.